src/main/java/de/retest/recheck/cli/utils/FilterUtil.java

```diff
@@ -75,4 +83,31 @@ public static boolean hasValidExcludeOption( final List<String> exclude ) {
 		return true;
 	}
 
+	public static List<Path> getFilterPaths( final Filter filterFiles ) {
+		final List<Path> filterPaths = new ArrayList<>();
+
+		// path of ignore files is provided by loadRecheckIgnore(), cannot be cast to CompoundFilter
+		if ( filterFiles instanceof GlobalIgnoreApplier ) {
+			return filterPaths;
```

Comment:
Maybe use an `Optional<List<Path>>` so that on the other end it is clear that there is a distinction between an empty list and no list at all.